python/cudf/cudf/core/column/column.py

```diff
@@ -1239,6 +1239,13 @@ def _process_for_reduction(
             )
         return result_col
 
+    def _resolve_reduction_dtype(self, reduction_op: str) -> Dtype:
```

Comment:
Style nit: none of our methods follow a  `resolve_*()` naming convention, so it feels kind of out of place. Perhaps just  `_reduction_result_dtype`?

Reply:
Updated